app/src/main/java/com/simplemobiletools/calculator/activities/SimpleActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.simplemobiletools.calculator.activities
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.MenuItem
+
+import com.simplemobiletools.calculator.Config
+import com.simplemobiletools.calculator.R
+
+open class SimpleActivity : AppCompatActivity() {
```

Comment:
you can clear this class, leave only
"open class SimpleActivity : BaseSimpleActivity() {
}
"